### ISSUE TITLE
Extra linebreak (a blank line) between figure and caption

### ISSUE THREAD

**Issue by QiangF** [kjambunathan/org-mode-ox-odt#63]
It is introduced after ox-odt-9.2.6.256. Probably in ox-odt-9.2.6.258.
For example:

    #+caption: org-mode unicorn
    #+attr_odt: :width 6.1
    [[./samples/org-mode-unicorn.png]]

**Comment by kjambunathan** [comment]
> Extra linebreak (a blank line) between figure and caption 

If I export this,

```
#+caption: org-mode unicorn
#+attr_odt: :width 6.1
[[./org-mode-unicorn.png]]

There is a line-\\
break here.

```

I get this

![Screenshot from 2019-12-02 22-15-08](https://user-images.githubusercontent.com/19869864/69977914-a56ade00-1551-11ea-97ee-4ae9da9c8fb4.png)

There is no line break at all.  The clearance / space comes from the margin of the `Figure` paragraph style.  If I set the top margin to zero, I would expect LibreOffice to remove the space. But it doesn't actually do so.  I am sure this is a bug in LibreOffice.  You may want to take it up with LibreOffice team.

**Comment by QiangF** [comment]
But even if you change the margin to 0, there still seems to be too much empty space.
Release 9.2.6.256 doesn't have the problem, difference of ox-odt.el shows the changes are in

    (DEFUN ORG-ODT--RENDER-IMAGE/FORMULA (CFG-KEY HREF WIDTHS HEIGHTS &OPTIONAL))

In 9.2.6.256 it is

    ("CAPTIONEDAS-CHARIMAGE"
    ("ORGCAPTIONEDIMAGE"
    " STYLE:REL-WIDTH=\"100%\" STYLE:REL-HEIGHT=\"SCALE\"" "PARAGRAPH")
    ("ORGINLINEIMAGE" NIL "AS-CHAR"))

    ("CAPTIONEDPARAGRAPHIMAGE"
    ("ORGCAPTIONEDIMAGE"
    " STYLE:REL-WIDTH=\"100%\" STYLE:REL-HEIGHT=\"SCALE\"" "AS-CHAR")
    ("ORGIMAGECAPTIONFRAME" NIL "PARAGRAPH"))

In 9.2.6.258 it is

    ("CAPTIONEDAS-CHARIMAGE"
    ("ORGDISPLAYIMAGE"
    " STYLE:REL-WIDTH=\"100%\" STYLE:REL-HEIGHT=\"SCALE\"" "PARAGRAPH")
    ("ORGINLINEIMAGE" NIL "AS-CHAR"))

    ("CAPTIONEDPARAGRAPHIMAGE"
    ("ORGDISPLAYIMAGE"
    " STYLE:REL-WIDTH=\"100%\" STYLE:REL-HEIGHT=\"SCALE\"" "PARAGRAPH")
    ("ORGIMAGECAPTIONFRAME" NIL "PARAGRAPH"))

**Comment by QiangF** [comment]
The problem seems disappeared when I change CaptionedParagraphImage as

    ("CaptionedParagraphImage"
    ("OrgDisplayImage"
    " style:rel-width=\"100%\" style:rel-height=\"scale\"" "as-char")
    ("OrgImageCaptionFrame" nil "paragraph"))

**Comment by kjambunathan** [comment]
> Extra linebreak (a blank line) between figure and caption

Fixed in ox-odt-9.2.6.261.tar.

When I export this 

```
#+caption: org-mode unicorn
#+attr_odt: :width 3
[[./org-mode-unicorn.png]]

There is a line-\\
break here.

#+attr_odt: :width (3 7)
#+caption: This is a very long long long long long long long long caption.
[[./org-mode-unicorn.png]]

```

I get 

![Screenshot from 2019-12-03 08-55-16](https://user-images.githubusercontent.com/19869864/70018363-e099fb00-15ab-11ea-9a52-db5305a99585.png)
